### DISCUSSION TITLE
Map Custom Domains To Individual Mailboxes

### DISCUSSION THREAD

**Discussion by collide-vad** [simple-login/app#810] [Feature Requests]
Not sure if this can be done already. 

I would like to see a feature where I can map a custom domain that forwards email to a particular mailbox. For example if I create a domain name called "formyuse.com" can I map that domain and all its catch all aliases to a particular mailbox? so no matter what aliases I create using the domain "formyuse.com" - all emails get forwarded to the right mailbox.

I have multiple domains hosted at SL and I would like to reserve domains for each of my family member. 

domain 1 -> mailbox 1
domain 2->  mailbox 2
domain 3-> mailbox 3

Thanks

**Comment by thelittlerocket** [comment]
Under the custom domain tab, you can setting the default mailbox for that domain.
Will effect aliases created after the change for that domain.

**Reply by collide-vad** [reply]
awesome.. that does exactly what I wanted.. thanks
